components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -335,11 +335,14 @@ sealed class ContentAction : BrowserAction() {
         ContentAction()
 
     /**
-     * Removes the [DownloadState] of the [ContentState] with the given [sessionId].
+     * Closes the [DownloadState.response] of the [ContentState.download] with the given [sessionId].
      */
-    data class ConsumeDownloadAction(val sessionId: String, val downloadId: String) :
-        ContentAction()
+    data class CloseDownloadResponse(val sessionId: String) : ContentAction()
```

Comment:
nit: Let's keep the *Action in the name to keep this unified.

Reply:
Sorry, it was unintentional I will add it.